source/index.html.md

```diff
@@ -673,6 +673,38 @@ object][ApplePay-PaymentToken] for more information.
   strongly authenticated (SCA in PSD2).
 </p>
 
+##### Method: `token`
+
+<dl class="dl-vertical">
+  <dt>token[tpan]
```

Comment:
Perhaps just `token[pan]` as it is evident from `token` that it is a token PAN? :man_shrugging: 

The same for `token[tav]` (`token[av]`).